handlers.go

```diff
@@ -705,11 +727,12 @@ func (cm *ContentManager) addDatabaseTrackingToContent(ctx context.Context, cont
 
 		return node.Links(), nil
 	}, root, cset.Visit, merkledag.Concurrent())
+
 	if err != nil {
 		return err
 	}
 
-	if err := cm.addObjectsToDatabase(ctx, cont, objects, "local"); err != nil {
+	if err = cm.addObjectsToDatabase(ctx, cont, objects, "local"); err != nil {
```

Comment:
lol, we should always rescope errors when doing a compound if statement. Ensures we dont ever accidentally use the wrong error value